test/unit/identity/storage/WrappedFetchAdapterFactory.test.ts

```diff
@@ -1,36 +1,40 @@
-import { literal, namedNode, quad } from '@rdfjs/data-model';
-import fetch from '@rdfjs/fetch';
-import type { DatasetResponse } from '@rdfjs/fetch-lite';
+import fetch from 'cross-fetch';
 import type { Adapter } from 'oidc-provider';
-import type { Dataset, Quad, Term } from 'rdf-js';
 import type { AdapterFactory } from '../../../../src/identity/storage/AdapterFactory';
 import { WrappedFetchAdapterFactory } from '../../../../src/identity/storage/WrappedFetchAdapterFactory';
-import { SOLID } from '../../../../src/util/Vocabularies';
+import { RdfToQuadConverter } from '../../../../src/storage/conversion/RdfToQuadConverter';
 
-jest.mock('@rdfjs/fetch');
```

Comment:
Can we drop the dependency?

Reply:
It's still used in the `TokenOwnershipValidator`. Although we probably want to replace it there as well at some point.